cmd/new.go

```diff
@@ -5,29 +5,36 @@
 package cmd
 
 import (
-	"errors"
-
+	"bytes"
+	"fmt"
 	"github.com/elastic/elastic-agent-changelog-tool/internal/changelog/fragment"
 	"github.com/spf13/afero"
 	"github.com/spf13/cobra"
 	"github.com/spf13/viper"
+	"os/exec"
 )
 
-var errNewCmdMissingArg = errors.New("new requires title argument")
-
 func NewCmd() *cobra.Command {
 
 	newCmd := &cobra.Command{
 		Use:   "new title",
 		Short: "Create a new changelog fragment",
 		Args: func(cmd *cobra.Command, args []string) error {
-			if len(args) < 1 {
-				return errNewCmdMissingArg
-			}
 			return nil
 		},
 		RunE: func(cmd *cobra.Command, args []string) error {
-			title := args[0]
+			title, err := cmd.Flags().GetString("title")
```

Comment:
@ycombinator the only drawback I see in this change is that it introduces a breaking change, as the title is now a flag and not an argument. Due to this the current documented usage `new <title>` breaks.

I would either change the changelog fragment kind to `breaking-change` or change the code to use `args[0]` like before (my preferred choice).

Reply:
I've changed the behavior to consider the string after `elastic-agent-changelog-tool new` to be the title.